db/migrate/20220520211950_create_users.rb

```diff
@@ -0,0 +1,9 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+
```

Comment:
The users' table should have a column name `photo` of `string` data type, `bio` of `text` data type, and `posts_counter` of `integer` data type.